components/sysutils/glances/glances.p5m

```diff
@@ -0,0 +1,267 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2022 Benny Lyons
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.human-version value=$(HUMAN_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file path=usr/bin/glances
+file path=usr/lib/python$(PYVER)/vendor-packages/Glances-$(COMPONENT_VERSION)-py$(PYVER).egg-info/PKG-INFO
```

Comment:
sample-manifest.p5m differs here: it contains usr/bin/glances-$(PYVER)

Reply:
To get `usr/bin/glances` in `sample-manifest.p5m` too you need to add `COMPONENT_POST_INSTALL_ACTION=` to `Makefile` after the `common.mk` include.